### PR TITLE
Add some style

### PR DESCRIPTION
**Pull request by Sabri-na00**
Pull request for the search field, search filter on dump-bag list of databases.

### CODE REVIEW

**Comment by chroi** [comment]
Hi @Sabri-na00 👋🏻

I think you're missing the git pre-commit hooks that lint the code. Did you check with Sirak to install them?

File: server/dumpbagserver/static/css/odoo-dump-bag.css [hunk 1]
```diff
@@ -34,17 +61,57 @@
...
+     your background is light. */
+  color: white;
+}
+
+.mdl-layout__header{
+ height: 6.5em;
+ background-color: white;
+}
+
+.logo-camp-to-camp{
```
Comment (guewen) [inline server/dumpbagserver/static/css/odoo-dump-bag.css:97]: This is a public project, we should keep it generic

File: server/dumpbagserver/views.py [hunk 2]
```diff
@@ -5,55 +5,73 @@
...
+                if sname in dbs:
+                    res[dbs] = dumps[dbs]
+            flash("There is a dababase by the name of", sname)
+            return render_template(
+                "dumps.html",
+                result=res,
+                download_commands=bagger.download_commands,
+                form=search_form,
+            )
+    return render_template(
```
Comment (mmequignon) [inline server/dumpbagserver/views.py:52]: Should GET and PUT methods return the same html ?

File: server/dumpbagserver/views.py [hunk 3]
```diff
@@ -5,55 +5,73 @@
...
 
-@app.route('/')
+@app.route("/", methods=["GET", "POST", "PUT"])
 def dumps():
     bagger = Bagger(app_config)
+    search_form = SearchForm(request.form)
     dumps = bagger.list_dumps()
-    return render_template('dumps.html', dumps=dumps,
-                           download_commands=bagger.download_commands)
+    if request.method == "POST":
```
Comment (mmequignon) [inline server/dumpbagserver/views.py:38]: Would it not be better to add a separate method to handle the POST method ?

File: server/dumpbagserver/config.py [hunk 4]
```diff
@@ -9,7 +9,7 @@
 
 class FlaskConfig(object):
     SECRET_KEY = env.get('FLASK_SECRET_KEY', '')
-    DEBUG = env.get('FLASK_DEBUG', False)
+    DEBUG = env.get('FLASK_DEBUG', True)
```
Comment (mmequignon) [inline server/dumpbagserver/config.py:12]: Is it wanted in production ?

File: server/dumpbagserver/templates/dumps.html [hunk 5]
```diff
@@ -2,70 +2,118 @@
...
+                 <pre>{{download_commands(database, dump)}}</pre>
+                 <a
+                   title="Download {{dump}}"
+                   class="mdl-button mdl-js-button mdl-button--colored mdl-button--raised mdl-js-ripple-effect"
+                   href="{{ url_for('download_dump', db=database, filename=dump) }}"
+                   >
+                   Download
+                 </a>
+               </span>
+             </li>
```
Comment (guewen) [inline server/dumpbagserver/templates/dumps.html:88]: This whole block is duplicated. Maybe use another template included in the 2 places for this block?

**Comment by guewen** [comment]
Replaced by #17

**Comment by guewen** [comment]
> Replaced by #17

Or is it? Can you reopen with only the styles part if needed?

**Pull request closed without merging**